src/styles/helpers/color/colorToRgba.js

```diff
@@ -0,0 +1,22 @@
+import decomposeColor from './decomposeColor';
+import rgbToRgba from './rgbToRgba';
+import hslToRgb from './hslToRgb';
+import { recomposeColor } from '.';
+
+export default function colorToRgba(color) {
+    if (typeof color !== 'string') return '';
+
+    try {
+        const { type, values } = decomposeColor(color);
+
+        if (!['rgb', 'rgba', 'hsl', 'hsla'].includes(type)) return '';
+
+        if (type === 'rgb') return rgbToRgba(color);
+        if (type === 'hsl') return rgbToRgba(hslToRgb(color));
+        if (type === 'hsla') return hslToRgb(color);
+
+        return recomposeColor({ type, values });
```

Comment:
I assume that this is the rgba case, is not better to return the color directly without need to pass through recomposeColor?